src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/RoleAssignmentTests.ps1

```diff
@@ -482,17 +482,9 @@ function Test-RaPropertiesValidation
     $roleDef.Description = "Read, monitor and restart virtual machines"
     $roleDef.AssignableScopes[0] = "/subscriptions/4004a9fd-d58e-48dc-aeb2-4a4aec58606f"
 
-    [Microsoft.Azure.Commands.Resources.Models.Authorization.AuthorizationClient]::RoleDefinitionNames.Enqueue("032F61D2-ED09-40C9-8657-26A273DA7BAE")
+    [Microsoft.Azure.Commands.Resources.Models.Authorization.AuthorizationClient]::RoleDefinitionNames.Enqueue("171d2453-29dc-42b4-84fc-3fd259eeaec3")
     New-AzureRmRoleDefinition -Role $roleDef
     $rd = Get-AzureRmRoleDefinition -Name "Custom Reader Test"
-
-    # Test
-    [Microsoft.Azure.Commands.Resources.Models.Authorization.AuthorizationClient]::RoleAssignmentNames.Enqueue("2fec1202-19ad-4c8b-b461-6031a1b5dce6")
-    $newAssignment = New-AzureRmRoleAssignment `
```

Comment:
@darshanhs90 why is this command being removed from this test?

Reply:
it was a redundant thing added in the test,as you can see earlier in the same test,we are creating a roleassignment, so that would be sufficient to validate the scenario for this test.

Reply:
my bad,thought the earlier create was also for a roleassignment, will revert it